types/node/crypto.d.ts

```diff
@@ -556,9 +556,6 @@ declare module 'crypto' {
          */
         namedCurve?: string | undefined;
     }
-    interface JwkKeyExportOptions {
```

Comment:
This is also duplicated in `v16` directory, we can remove that too.

Reply:
Done